### ISSUE TITLE
Grabber crashes when going from page 26 to 25 on Sankaku

### ISSUE THREAD

**Issue by jukebox1412** [Bionus/imgbrd-grabber#1299]
### What steps will reproduce the problem?
1. Find some user with more than 26 pages of favorites on chan.sankaku and search using their username e.g. 'fav:jukebox1412' NSFW warning

2. Go to page 26 of the search results (which is also broken and only shows the top 4 most popular) 
![image](https://user-images.githubusercontent.com/22598492/41330364-400703c2-6e90-11e8-9e7c-4520f9aba2dd.png)

3. Go to page 25
![image](https://user-images.githubusercontent.com/22598492/41330377-51028fb6-6e90-11e8-9c1a-b203ff91315d.png)

### What is the expected behavior? What do you get instead?
I expect it to go to page 25. It crashes instead.

### How often does this problem occur? On which sources?
Always. Other sources are not reproducible as I do not have an account on other image boards.

### What version of the program are you using? On what operating system?
6.0.2 Windows 10 x64

### Please provide any additional information below
I've noticed that when going to page 25 and above for favorites in Sankaku, the only images loaded are the top 4 most popular images at that time. Following the link 
![image](https://user-images.githubusercontent.com/22598492/41330418-a1f2ce4a-6e90-11e8-8c56-a5ba39ffdf3b.png)
to Sankaku will lead me to a url such as this:

_NSFW WARNING_
https://chan.sankakucomplex.com/post/index?next=5190595&tags=fav:jukebox1412%20-video

However, if I replace the 'next' attribute to 6514530 instead of 5190595, it displays the correct images. I got this from going into Sankaku's website and actually clicking the next button to get to page 25 from page 24.
https://chan.sankakucomplex.com/post/index?next=6514530&tags=fav:jukebox1412%20-video

Also, if I choose to use the URL https://chan.sankakucomplex.com/?tags=fav%3Ajukebox1412&page=26 _while logged in to the site_ I can avoid the `next` attribute on browser. However, I this only works if I'm logged into to the site.

**Comment by jukebox1412** [comment]
Also there's a workaround to the second issue I mentioned, just put &next=<#> into the tags and you'll be able to pull the page

**Comment by Bionus** [comment]
I fixed a bunch of issues with Sankaku paging system in 6.0.3, can you give it a try?

Also, if you're logged in the max page should be 50, not 25. Did you provide your credentials in Grabber source settings?

![image](https://user-images.githubusercontent.com/882719/41429788-6919b08c-700e-11e8-8999-1b13bfe2bcf6.png)

> However, if I replace the 'next' attribute to 6514530 instead of 5190595, it displays the correct images. I got this from going into Sankaku's website and actually clicking the next button to get to page 25 from page 24.

If you're going from a high page to its previous (for example 137 to 136) what will be set is the `prev` parameter. The `next` parameter is used when going from a low page to a higher one (for example 136 to 137).

> It crashes instead.

Can you reproduce this crash on v6.0.3 then upload your crash dump? You can find them in `C:/Users/%USERNAME%/AppData/Local/Bionus/Grabber/crashes` (note that I'm only interested in 6.0.3 dumps, not 6.0.2).

**Comment by jukebox1412** [comment]
Seems that the original problem has been fixed. 

Also with 6.0.3, I've found that I can get past page 25+ without using the &next=<number> trick now. Use some browser to log in to Sankaku and then get your cookies for login and passhash either from some addon (I used Opera's cookie manager) or by directly reading the cookies. Then add your cookies into Grabber and enter your login under the credentials tab, this should allow you to view past the first page for your own account's **fav:<user>**
